it/grpc/java/src/test/java/com/linecorp/armeria/grpc/java/ClientHttp2GracefulShutdownTimeoutOverridingTest.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.grpc.java;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Duration;
+import java.util.concurrent.CountDownLatch;
+
+import org.junit.jupiter.api.Test;
+
+import com.google.common.util.concurrent.ListenableFuture;
+
+import com.linecorp.armeria.client.ClientFactory;
+import com.linecorp.armeria.client.grpc.GrpcClients;
+import com.linecorp.armeria.grpc.java.Hello.HelloReply;
+import com.linecorp.armeria.grpc.java.Hello.HelloRequest;
+import com.linecorp.armeria.grpc.java.HelloServiceGrpc.HelloServiceFutureStub;
+
+import io.grpc.Server;
+import io.grpc.ServerBuilder;
+import io.grpc.stub.StreamObserver;
+
+class ClientHttp2GracefulShutdownTimeoutOverridingTest {
+
+    @Test
```

Comment:
Did you say that `@FlakyTest` is set to this?
I'm fine with either way.

Reply:
I thought it was okay because this test is now in the :it: module not in the :core: module which takes longer to complete. 😄